pairings/frontend/templates/frontend/base.html

```diff
@@ -25,6 +25,18 @@
   </head>
 
   <body>
+    <div class="container-lg mt-3">
+        {% if messages %}
+            {% for message in messages %}
+            <div class="alert alert-danger border border-danger shadow-sm alert-dismissible fade show" role="alert">
```

Comment:
This only allows for the error messages -- change the template to properly map the message type to the bootstrap style.

messages.error -> danger
messages.warning -> warning

etc.